CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- [react] feat: Export `createReduxEnhancer` to log redux actions as breadcrumbs, and attach state as an extra. (#2717)
```

Comment:
This usually goes below the last CHANGELOG item, so in this case below `[tracing] fix: Add manual `DOMStringList` typing (#2718)`